stargazer.js

```diff
@@ -0,0 +1,67 @@
+import Alt from './'
+import { createStore, datasource } from './utils/decorators'
+import axios from 'axios'
+
+const alt = new Alt()
+
+const StargazerActions = alt.generateActions(
+  'fetchingUsers',
+  'usersReceived',
+  'failed'
+)
+
+const StargazerSource = {
```

Comment:
Using the datasource decorator (or the equivalent store config option), a user could implement a DefaultSource class like you are suggesting.  They could then provide whatever default behavior they desire, and override in any child classes.  So what you are suggesting is possible, but should not be part of alt itself IMO.